e2e_tests/tests/createAndRemove.spec.ts

```diff
@@ -0,0 +1,137 @@
+import { test, expect } from "@playwright/test";
+
+test.setTimeout(5000);
+
+const createAndRemove = [
+	{ type: "sidebar", locator: `div.CoreSidebar.component` },
+	{ type: "section", locator: `section.CoreSection.component` },
+	{ type: "columns", locator: `div.CoreColumns.component` },
+];
+
+const fullCheck = [
+	{ type: "button", locator: `button.CoreButton.component` },
+	{ type: "text", locator: `div.CoreText.component` },
+	{ type: "header", locator: `div.CoreHeader.component` },
+	{ type: "heading", locator: `div.CoreHeading.component` },
+	{ type: "dataframe", locator: `div.CoreDataframe.component` },
+	{ type: "html", locator: `div.CoreHTML.component` },
+	{ type: "pagination", locator: `div.pagination.component` },
+	{ type: "repeater", locator: `div.CoreRepeater.component` },
+	//{ type: "column", locator: `div.CoreColumn.component` },
+	//{ type: "tab", locator: `div.CoreTab.component` },
+	{ type: "tabs", locator: `div.CoreTabs.component` },
+	{ type: "horizontalstack", locator: `div.CoreHorizontalStack.component` },
+	{ type: "separator", locator: `div.CoreSeparator.component` },
+	{ type: "image", locator: `div.CoreImage.component` },
+	{ type: "pdf", locator: `div.CorePDF.component` },
+	{ type: "iframe", locator: `div.CoreIFrame.component` },
+	{ type: "googlemaps", locator: `div.CoreGoogleMaps.component` },
+	{ type: "icon", locator: `div.icon.component` },
+	{ type: "timer", locator: `div.CoreTimer.component` },
+	{ type: "textinput", locator: `div.CoreTextInput.component` },
+	{ type: "textareainput", locator: `div.CoreTextareaInput.component` },
+	{ type: "numberinput", locator: `div.CoreNumberInput.component` },
+	{ type: "sliderinput", locator: `div.CoreSliderInput.component` },
+	{ type: "dateinput", locator: `div.CoreDateInput.component` },
+	{ type: "radioinput", locator: `div.CoreRadioInput.component` },
+	{ type: "checkboxinput", locator: `div.CoreCheckboxInput.component` },
+	{ type: "dropdowninput", locator: `div.CoreDropdownInput.component` },
+	{ type: "selectinput", locator: `div.CoreSelectInput.component` },
+	{ type: "multiselectinput", locator: `div.CoreMultiselectInput.component` },
+	{ type: "fileinput", locator: `div.CoreFileInput.component` },
+	{ type: "webcamcapture", locator: `div.CoreWebcamCapture.component` },
+	{ type: "vegalitechart", locator: `div.CoreVegaLiteChart.component` },
+	{ type: "plotlygraph", locator: `div.CorePlotlyGraph.component` },
+	{ type: "metric", locator: `div.CoreMetric.component` },
+	{ type: "message", locator: `div.CoreMessage.component` },
+	{ type: "videoplayer", locator: `div.CoreVideoPlayer.component` },
+];
```

Comment:
I have an idea, we should do a pass of the whole `templateMap` and check whether we have tests for everything mentioned there. If we don't, we fail.

Otherwise we can easily forget.

Or maybe some `--strict` flag so that we can pass a PR but not release?

Keen to hear your thoughts.

Reply:
I was thinking about using `templateMap` here earlier but I don't have access to this file from here. Thats why I just created list of components base on `templateMap`.

Reply:
If you have access to the `core` you can use `getSupportedComponentTypes` without doing an import of `templateMap` which I agree, wouldn't be the cleanest.